### DISCUSSION TITLE
Create new FL project from script?

### DISCUSSION THREAD

**Discussion by xanmankey** [demberto/PyFLP#169] [Q&A] [answered]
Haven't had time to comb through documentation or code yet; I know this is a parser for editing flp files, but is there a way to create a new flp project file from script? Also, impressive job on the parser considering FL is closed source!

Regards

**Comment by tochibedford** [comment, marked as answer]
PyFLP is purely for parsing and modifying (to a certain capacity) flp files, not creating them, at least as far as I’m aware. You can also re-save an FLP as a zip format using it if you like.

[Check here for zip exports](https://pyflp.readthedocs.io/en/latest/handbook.html#exporting-to-a-zip)

**Comment by demberto** [comment]
There's no correct cross version way to do this. A generated FLP would always work with a few versions of FL correctly and a few not. There are a bit too many unknowns to be doing this in some decent manner.

**Comment by xanmankey** [comment]
I see, makes sense. Thank you for the prompt response, I'll set about manually creating them in that case!
